gradle.properties

```diff
@@ -17,7 +17,7 @@ POM_DEVELOPER_ID=square
 POM_DEVELOPER_NAME=Square
 POM_DEVELOPER_URL=https://github.com/square/
 
-org.gradle.jvmargs=-Xmx2048m
+org.gradle.jvmargs=-Xmx4g
```

Comment:
With 1k modules that seemed to be a good choice. I ran a few Gradle scans and the it was using up to 2g of the heap.